centos/7/Dockerfile

```diff
@@ -6,6 +6,10 @@ ARG DEVTOOLSET=devtoolset-8
 # Fix missing locales
 ENV LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8"
 
+# Update repositories to avoid of issues got at:
```

Comment:
`yum/dnf update` updates repositories **and installed packages** (unlike `yum/dnf makecache`). The comment may mislead someone (especially, because `apt-get update` updates only repositories metadata).

Reply:
Sure, added.